apps/explorer/config/config.exs

```diff
@@ -161,7 +160,22 @@ config :explorer, Explorer.Counters.Bridge,
   update_interval_in_seconds: bridge_market_cap_update_interval || 30 * 60,
   disable_lp_tokens_in_market_cap: System.get_env("DISABLE_LP_TOKENS_IN_MARKET_CAP") == "true"
 
-config :explorer, Explorer.ExchangeRates, enabled: System.get_env("DISABLE_EXCHANGE_RATES") != "true", store: :ets
+config :explorer, Explorer.ExchangeRates,
+  enabled: System.get_env("DISABLE_EXCHANGE_RATES") != "true",
+  store: :ets,
+  coingecko_coin_id: System.get_env("EXCHANGE_RATES_COINGECKO_COIN_ID"),
```

Comment:
It will break, yes, the plan is to merge June before deploying it - there is a [fix](https://github.com/blockscout/blockscout/commit/4886204f27cb05b1e7f8d8c5fa796705dd9fbf9c#diff-6e594c281f80f6246e0825fca922f0c4403ae4b0730cde70f58d901b7c99ce79) that addresses it. I don't like it but it'll be less effort that way.

Reply:
Meanwhile, switched to free API.